pkg/operator/options.go

```diff
@@ -97,18 +99,21 @@ func NewOptionsWithManagerOrDie() (Options, manager.Manager) {
 	}
 
 	manager := NewManagerOrDie(ctx, controllerRuntimeConfig, opts)
-	recorder := events.NewRecorder(manager.GetEventRecorderFor(appName))
+
+	baseRecorder := manager.GetEventRecorderFor(appName)
```

Comment:
🤔 , if our EventRecorder subclassed this recorder, we wouldn't have this encapsulation problem. Happy to punt later. Can you cut an OE issue to clean up?